vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -127,214 +149,168 @@ private void initTagAndNotesLayout() {
-    }
+  private void loadExperimentInfo(Experiment experiment) {
+    ExperimentInfoComponent factSheet = ExperimentInfoComponent.create(experiment.getSpecies(),
+        experiment.getSpecimens(), experiment.getAnalytes());
+    ExperimentalVariablesComponent experimentalVariables = ExperimentalVariablesComponent.create(
+        experiment.variables());
+    ExperimentDetailsComponent.this.experimentSummary.add(factSheet);
+    ExperimentDetailsComponent.this.experimentSummary.add(experimentalVariables);
+    factSheet.showMenu();
+  }
 
-    private void displayDisclaimer() {
-      contentExperimentalGroupsTab.add(noExperimentalVariablesDefined);
-    }
+  private void loadBlockingVariableInformation() {
+    //ToDo load information from backend once implemented
+  }
 
-    private void removeDisclaimer() {
-      contentExperimentalGroupsTab.remove(noExperimentalVariablesDefined);
-    }
+  private void fillExperimentalGroupDialog() {
+    Objects.requireNonNull(experimentId, "experiment id not set");
+    List<ExperimentalVariable> variables = experimentInformationService.getVariablesOfExperiment(
+        experimentId);
+    List<VariableLevel> levels = variables.stream()
+        .flatMap(variable -> variable.levels().stream()).toList();
+    experimentalGroupsDialog.setLevels(levels);
+  }
 
-    private void subscribeToDeletionClickEvent(ExperimentalGroupCard experimentalGroupCard) {
-      experimentalGroupCard.addDeletionEventListener(ExperimentDetailsComponent.this::handleEvent);
-    }
+  private void displayDisclaimer() {
+    contentExperimentalGroupsTab.add(noExperimentalVariablesDefined);
+  }
 
-    public void onGroupSubmitted(ExperimentalGroupSubmitEvent groupSubmitted) {
-      Result<ExperimentalGroup, ResponseCode> response = experimentInformationService.addExperimentalGroupToExperiment(
-          experimentId,
-          new ExperimentalGroupDTO(groupSubmitted.variableLevels(), groupSubmitted.sampleSize()));
-      if (response.isValue()) {
-        handleGroupSubmittedSuccess();
-      } else {
-        handleDuplicateConditionInput();
-      }
-    }
+  private void hideExperimentalGroupsCollection() {
+    contentExperimentalGroupsTab.remove(experimentalGroupsCollection);
+  }
 
-    private void handleGroupSubmittedSuccess() {
-      loadExperimentalGroups();
-      addCreationCard();
-      experimentalGroupsDialog.close();
-    }
+  private void removeDisclaimer() {
+    contentExperimentalGroupsTab.remove(noExperimentalVariablesDefined);
+  }
 
-    private void handleDuplicateConditionInput() {
-      InformationMessage infoMessage = new InformationMessage(
-          "A group with the same condition exists already.", "");
-      StyledNotification notification = new StyledNotification(infoMessage);
-      notification.open();
-    }
+  private void displayExperimentalGroupsCollection() {
+    contentExperimentalGroupsTab.add(experimentalGroupsCollection);
+  }
```

Comment:
NIT: Since you only trigger the display of the component, you could also keep it added to the parent component and switching it's visibility via the `setVisible(boolean visibility) method`